api.js

```diff
@@ -67,6 +69,13 @@ const mapRouteParsers = (route, parsers) => {
 			defaultStr: '*adult*',
 			parse: parseInteger
 		},
+		routingMode: {
+			description: 'HAFAS routing mode, see [Routing Mode](#routing-mode)',
```

Comment:
The link won't work in the OpenAPI spec, which is generated from this markup. What do you think about referring to the section without linking to it?

```suggestion
			description: 'HAFAS routing mode, see the "Routing Mode" section',
```

Reply:
Ah right. I think it's good enough, yeah 👍